### ISSUE TITLE
Couldn't unselect the default template from the template dropdown menu

### ISSUE THREAD

**Issue by saifobeidat** [umbraco/Umbraco-CMS#6318]
I could not unselect the default template form the template dropdown menu, there's no "choose" option anymore.

### Bug summary
We used to find "choose" option if you want to unselect the default template in Umbraco7, but I couldn't find it in Umbraco8

### Specifics

Umbraco : 8.0+

### Steps to reproduce
1- Create a docymenttype with a template, then create a node with that type
2- Go to that  node then and go to info tab
2- There's no "choose" option in the template dropdown menu

### Expected result
 Cannot unselect the default template from the template dropdown menu

### Actual result
unable to remove the default template from a node

**Comment by kjac** [comment]
@saifobeidat thanks for reporting 👍 however I'm pretty sure this is by design. Can you describe the use case for wanting to *remove* the template from a piece of content?

**Comment by saifobeidat** [comment]
@kjac ,

In our case, we are displaying a short introduction text in the website, this text might get bigger in the future, so we have provided a "Read More" button that leads you to an inner page, but this button will only appear when our Umbraco user decides to select a template for that introduction text.

**Comment by kjac** [comment]
@saifobeidat sounds like a slightly convoluted editing experience; wouldn't it be an option to make this an explicit editor setting/configuration on the outer page? 

The reason I ask all this is that this is the first time I have heard of people wanting to *unset* the content template. I'm trying to figure out if this is indeed a meaningful operation (read: something that should be possible to do in the UI).

**Comment by saifobeidat** [comment]
@kjac 

Yes we can create an option to hide/show the "Read More" button which leads to outer page, but in this case we would still have a template for that content, what if someone could open that outer page by writing its URL? he would find a page with inappropriate short text (not ready with full content). but if we had that content without template, he would be redirected to another page (which we can control). 

so that's why we need our Umbraco user to be able to unset the content's template.

It's not just for that use, sometimes we use it when we build a website menu, we check if a certain node doesn't have a template (not selected) then we might hide it or do any action about it.

**Comment by kjac** [comment]
@saifobeidat If you don't want certain content visible but you're not ready to delete it either, you can always just unpublish it. 

Anyway I do get your point; I think @nul800sebastiaan will have to weigh in here. Your requirement might clash a bit with the automatic assignment of the default content template.

**Comment by stevemegson** [comment]
I've occasionally unset the template for content when I want a "placeholder" to give its children the desired URLs. The parent might just be a 404 or redirect, or I might have a custom MVC route handling the parent's URL.

However, I think that every time I've done that, I could instead have created a "placeholder" template called None, Hidden, Blank, or whatever else makes sense to editors. The name of the template can then more explicitly describe what happens when you select it, and the option can be made available only on document types where your code will check for it and do something sensible.

**Comment by saifobeidat** [comment]
> I've occasionally unset the template for content when I want a "placeholder" to give its children the desired URLs. The parent might just be a 404 or redirect, or I might have a custom MVC route handling the parent's URL.
> 
> However, I think that every time I've done that, I could instead have created a "placeholder" template called None, Hidden, Blank, or whatever else makes sense to editors. The name of the template can then more explicitly describe what happens when you select it, and the option can be made available only on document types where your code will check for it and do something sensible.

Yes exactly, we have done that already, we created a template called "No Template" to do what we want.
